director/director_ui.go

```diff
@@ -251,8 +251,14 @@ func handleFilterServer(ctx *gin.Context) {
 	// If we previously temporarily allowed a server, we switch to permFiltered (reset)
 	if filterType == tempAllowed {
 		filteredServers[sn] = permFiltered
+		if err := SetServerDowntime(sn, permFiltered); err != nil {
+			log.Errorln("Failed to persist the downtime in director db", err)
```

Comment:
These two error log lines should have a colon to separate the your error handle and the error produced by the function.